### ISSUE TITLE
Folder/group and "new group" buttons at the bottom require a double click to activate

### ISSUE THREAD

**Issue by leviport** [pop-os/cosmic-applibrary#8]
### Summary:
The categories/folders at the bottom of the launcher only become selected when I single-click them. They are activated by either a double click, or by hitting Enter after one is selected.

### Steps to reproduce: 
- Super + A to open library
- Single click one of the groups at the bottom or the "new group" button, it won't activate
- Double click one of the groups at the bottom or the "new group" button, and it does activate

### Expected behavior:
I'm thinking a single click should probably activate these buttons.

### Version:
```
$ apt policy cosmic-app-library 
cosmic-app-library:
  Installed: 0.1.0~1660784434~22.04~dc85e14
  Candidate: 0.1.0~1660784434~22.04~dc85e14
  Version table:
 *** 0.1.0~1660784434~22.04~dc85e14 1002
       1001 http://apt.pop-os.org/release jammy/main amd64 Packages
       1002 http://apt.pop-os.org/staging/master jammy/main amd64 Packages
        100 /var/lib/dpkg/status
```

**Comment by wash2** [comment]
this should be fixed by bedfe1da354e1b32a1af2df9ed252e5dcaaf376a

**Comment by n3m0-22** [comment]
This is working now. Single click behavior for groups and new group is working correctly.
